tests/models/imagegpt/test_modeling_imagegpt.py

```diff
@@ -108,7 +99,7 @@ def get_large_model_config(self):
     def prepare_config_and_inputs(
         self, gradient_checkpointing=False, scale_attn_by_inverse_layer_idx=False, reorder_and_upcast_attn=False
     ):
-        pixel_values = ids_tensor([self.batch_size, self.seq_length], self.vocab_size - 1)
```

Comment:
`pixel_values` -> `input_ids`

Many tests in this model were overwritten because `input_ids` were part of the model input dictionary as `pixel_values`. With this change we can delete these overwrites. And, more importantly, unblock further test changes without requiring overwrites.